python/lsst/dax/imgserv/image.py

```diff
@@ -0,0 +1,118 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+"""
+This module implements the Image request data model and operations.
+
+@author: Kenny Lo, SLAC
+
+"""
+
+class Image(object):
+    """ Image module maps a request and its parameters per JSON schema to the 
+        corresponding imagegetter method. 
+    """
+  
+    # class variable (static)
+    imagegetter = None
```

Comment:
I've redone the work to make imagegetter instance as parameter to be passed in instead.